_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -376,11 +376,44 @@ These two HTML `a` elements have the same [accessible name][] and link to the [s
 
 ```html
 <html lang="en">
-	<div>You can learn more in the <a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/index.html">Contact us</a> page.</div>
-	<div>You can find contact information in the <a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/index.html">Contact us</a> page.</div>
+	<div>
+		You can learn more in the
+		<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/index.html">Contact us</a> page.
+	</div>
+	<div>
+		You can find contact information in the
+		<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/index.html">Contact us</a> page.
+	</div>
 </html>
 ```
 
+#### Inapplicable Example 6
+
+These lists have `a` elements with the same [accessible names][accessible name] but different [programmatically determined link contexts][programmatically determined link context] as determined by their preceding `h2` elements.
+
+```html
+<h2>ACT at WAI</h2>
+<ul>
+	<li><a href="https://www.w3.org/WAI/standards-guidelines/act/">Overview</a></li>
+	<li><a href="https://www.w3.org/WAI/standards-guidelines/act/rules/">List of rules</a></li>
+</ul>
+
+<h2>ACT at GitHub</h2>
+<ul>
+	<li><a href="https://act-rules.github.io/pages/about">Overview</a></li>
+	<li><a href="https://act-rules.github.io/rules/">List of rules</a></li>
+</ul>
+```
+
+#### Inapplicable Example 7
+
+These two HTML `a` elements have the same [accessible name][] but different [programmatically determined link contexts][programmatically determined link context] because the `div` elements place them in different display blocks.
+
+```html
+<div><a href="https://www.w3.org/WAI/">Read more</a> about the W3C WAI</div>
+<div><a href="https://www.w3.org/International/">Read more</a> about the W3C internationalization</div>
+```
```

Comment:
A bit sidetracking, but I feel like this ties to #1784 🤔 
Notably, if we restrict the rule to "same" (which is implied by Inapplicable Example 7 description), then the headings are unnecessarily in Inapplicable Example 6.
Which also sort of implies (backtracking my previous review 🙈 ) that the rule might not need to consider headings since I'm not sure that links can be in the **same** context but with different heading (except maybe with some heavy CSS  wrangling to avoid creating new blocks before and after headings).

Reply:
But see also https://github.com/act-rules/act-rules.github.io/issues/1784#issuecomment-1029046782
I'm not sure any more where I stand on this issue 🙃

Reply:
@jym I'm going to remove headings from the programatically determined context. We can't say that WCAG supports it. 
From WCAG 2.0 to WCAG 2.1, [headings were removed from F63](https://github.com/w3c/wcag/pull/832). 
WCAG2.0 - https://www.w3.org/TR/WCAG20-TECHS/F63.html
WCAG2.1 - https://www.w3.org/WAI/WCAG21/Techniques/failures/F63
Further, H80 moved from sufficient to advisory. So, all the evidence points against headings being part of programatically determined context.